custom_components/tahoma/water_heater_devices/domestic_hot_water_production.py

```diff
@@ -11,14 +11,20 @@
 
 from ..entity import OverkizEntity
 
-STATE_MANUAL = "manual"
-STATE_AUTO = "auto"
+DHWP_TYPE_MURAL = "io:AtlanticDomesticHotWaterProductionV2_MURAL_IOComponent"
+DHWP_TYPE_CE_FLAT_C2 = "io:AtlanticDomesticHotWaterProductionV2_CE_FLAT_C2_IOComponent"
+DHWP_TYPE_MBL = "modbuslink:AtlanticDomesticHotWaterProductionMBLComponent"
+
+STATE_AUTO = "Auto"
+STATE_BOOST = "Boost"
+STATE_MANUAL = "Manual"
 
```

Comment:
For the core PR, we should make them lowercase again and add them to translations. However I tried to do that in the past, and I couldn't get it to work :D.

Reply:
For Boost mode we could use STATE_HIGH_DEMAND or STATE_PERFORMANCE from core. 
Other modes are tricky to map directly:
- There is no auto mode in core.   
- MANUAL_ECO_ACTIVE can also be STATE_HEAT_PUMP from core.
- MANUAL_ECO_INACTIVE could be STATE_ELECTRIC from core, but for my water heater it doesnt mean only electric but electric and/or heat pump.
So the better option will be mimic Cozytouch names if it's allowed in core.

Reply:
For now, I think we can keep it like this. They are quite self explanatory.